unleash-yggdrasil/src/strategy_parsing.rs

```diff
@@ -332,7 +332,7 @@ fn rollout_constraint(mut node: Pairs<Rule>) -> RuleFragment {
 
 fn get_hostname() -> Result<String, SdkError> {
     //This is primarily for testing purposes
-    if let Ok(hostname_env) = env::var("HOSTNAME") {
+    if let Ok(hostname_env) = env::var("hostname") {
```

Comment:
Previously HOSTNAME was n arbitrary choice but this is a more consistent way of checking this, since this is what's actually set by most OSes

Reply:
Should we do "hostname" or else "HOSTNAME"?